openmc/capi/cell.py

```diff
@@ -114,6 +118,12 @@ def fill(self):
         else:
             raise NotImplementedError
 
+    @property
+    def temperature(self, instance=None):
```

Comment:
I like the idea of having two `capi` functions - it might be unexpected behavior for the user to call `openmc_cell_get_temperature` and receive an averaged temperature when they simply forgot to include an instance.

Reply:
Just checking that I'm up to date on where the conversations have taken us - @pshriwise, are you still wanting the `openmc_cell_get_temperature` method to not compute the average temp if no instance is passed (and have that instead be a separate method)?

Reply:
Yeah that was my thought. Or even just add `get_temperature` for now. Is there a current need for an averaged cell temperature?

Reply:
Not that I can think of - I originally included the average as some type of sensible calculation if an instance wasn't provided, but I'm not sure we need it either

Reply:
Maybe best to leave it out for now then. If one is using a model with distributed cell temperatures, hopefully they'll know they can gather them all via this function and compute an average if needed.